components/dot_menu/dot_menu.tsx

```diff
@@ -259,11 +259,12 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
     }
 
     handleEditMenuItemActivated = () => {
+        this.props.handleDropdownOpened?.(false);
         this.props.actions.setEditingPost(
             this.props.post.id,
             this.props.location === Locations.CENTER ? 'post_textbox' : 'reply_textbox',
             this.props.post.root_id ? Utils.localizeMessage('rhs_comment.comment', 'Comment') : Utils.localizeMessage('create_post.post', 'Post'),
-            this.props.location === Locations.RHS_ROOT || this.props.location === Locations.RHS_COMMENT,
+            this.props.location === Locations.RHS_ROOT || this.props.location === Locations.RHS_COMMENT || this.props.location === Locations.SEARCH,
```

Comment:
this was something that got pointed out during PR reviews ... maybe it is just that it is possible to do so at the moment and we don't want to miss that possibility? I am also not 100% sure that this is a good idea, but since it was possible before I added it in.

This would be a good question for @esethna and @matthewbirtch

Reply:
I would be good with that. I actually didn't realize I could edit posts from the search results list, but I would be fine with removing that ability (if we do this we should remove the ability to delete as well) from this context. Users would have to jump to the thread or channel to edit the post. @esethna do you think users use this much?

Alternatively, we could leave the ability to edit from the search results, but direct them to the post in the thread or channel with the post scrolled to and the inline edit input displayed. This sounds complicated though

Reply:
I am fine with both approaches. Keeping the ability to edit them in the search results just creates more opportunities and less jumping around for the user (maybe the user wants to edit all of his messages where he send a URL or something like that) .. there are definitely use-cases for that :D